.changes/unreleased/tinymce-TINY-11982-2025-05-07.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Fixed an issue where it wasn't possible to navigate out of a `figcaption` using the left and right arrow keys in Firefox.
```

Please don't put the word `fixed` in a fixed` kind
```suggestion
body: It wasn't possible to navigate out of a `figcaption` using the left and right arrow keys in Firefox.
```